### ISSUE TITLE
when parsing ipv6, get " ValueError: not enough values to unpack (expected 2, got 1)"

### ISSUE THREAD

**Issue by teto** [enkore/i3pystatus#440]
```
  File "/home/teto/i3pystatus/i3pystatus/core/modules.py", line 285, in __call__
    self.run()
  File "/home/teto/i3pystatus/i3pystatus/network.py", line 413, in run
    network_info = self.network_info.get_info(self.interface)
  File "/home/teto/i3pystatus/i3pystatus/network.py", line 105, in get_info
    format_dict.update(self.extract_network_info(network_info))
  File "/home/teto/i3pystatus/i3pystatus/network.py", line 121, in extract_network_info
    mask, bits = v6["netmask"].split("/")
ValueError: not enough values to unpack (expected 2, got 1)
```

Here is the result of 'ip addr'
```

1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host 
       valid_lft forever preferred_lft forever
2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP group default qlen 1000
    link/ether ac:9e:17:83:50:e9 brd ff:ff:ff:ff:ff:ff
    inet 192.168.1.12/24 brd 192.168.1.255 scope global dynamic eth0
       valid_lft 81803sec preferred_lft 81803sec
    inet6 2a01:XXXX:XXXX:XXXX:XXXX:XXXX:XXXX:XXXX/64 scope global temporary dynamic 
       valid_lft 847sec preferred_lft 247sec
    inet6 2a01:XXXX:XXXX:XXXX:XXXX:XXXX:XXXX:XXXX/64 scope global mngtmpaddr noprefixroute dynamic 
       valid_lft 847sec preferred_lft 247sec
    inet6 fe80::abe2:32e:2482:2f4f/64 scope link 
       valid_lft forever preferred_lft forever

```

**Comment by teto** [comment]
Can't we get the mask programmatically ? I would expect netifaces to be able to provide it without having to parse a string ? (mask, bits = v6["netmask"].split("/") )

**Comment by enkore** [comment]
Has this the same cause as #439?

Guess we need a new release asap (fix for #439)

**Comment by enkore** [comment]
Has this the same cause as #439?

Guess we need a new release asap (fix for #439)

**Comment by teto** [comment]
I don't think it's related, I updated this morning to git i3pystatus and then it triggered this issue:
My config doesn't use the upper_limit
```
status.register("network",
                format_up="{v4cidr}",
                hints= {"markup": "pango"},
                on_leftclick="ip addr show dev {interface} | xmessage -file -",
                )
```

**Comment by colajam93** [comment]
PR #435 (merged) depends on netifaces 0.10.5 and it include breaking backward compatibility.
Installing latest netifaces may fix this issue.

**Comment by teto** [comment]
Confirms upgrading netifaces fixes it (I wish so hard for pip to have an --upgrade-all command :) )
